src/renderer/components/dialog-bisect.tsx

```diff
@@ -145,4 +205,33 @@ export class BisectDialog extends React.Component<BisectDialogProps, BisectDialo
       </Dialog>
     );
   }
+
+  /**
+   * Should an item in the "earliest version" dropdown be disabled?
+   *
+   * @param {ElectronVersion} version
+   * @returns {boolean}
+   */
+  public isEarliestItemDisabled(version: ElectronVersion): boolean {
+    const { allVersions, endIndex } = this.state;
+
+    // In the array, "newer" versions will have a lower index.
+    // 0: 5.0.0
+    // 1: 4.0.0
+    // 2: 3.0.0
+    // ...
+    return allVersions.indexOf(version) < endIndex;
+  }
+
+  /**
+   * Should an item in the "latest version" dropdown be disabled?
+   *
+   * @param {ElectronVersion} version
+   * @returns {boolean}
+   */
+  public isLatestItemDisabled(version: ElectronVersion): boolean {
+    const { allVersions, startIndex } = this.state;
+
+    return allVersions.indexOf(version) > startIndex;
```

Comment:
```suggestion
    return allVersions.indexOf(version) > startIndex - 1;
```

Off by one -- current change still allows same version to be selected in both selectors.